### ISSUE TITLE
a relative path issue

### ISSUE THREAD

**Issue by mindon** [BjornTheProgrammer/bun-plugin-html#25]
https://github.com/BjornTheProgrammer/bun-plugin-html/blob/d621af22e8b4065f4be097cf0cc4c7f3c5984cdf/src/index.ts#L877

if there's a **hello/world.html** with 
```
<script type="module" src="../demo.ts"></script>
```
the output got 
```
<script type="module" src="./demo-xxxxxx.js">
```

**Comment by BjornTheProgrammer** [comment]
That makes sense, I'll try to implement a fix this week, just having finals currently. The rewrite has been delayed by the fact that a [copy loader](https://github.com/oven-sh/bun/issues/15587) doesn't exist. I've been working on that, but there really isn't a time span for when that might or will be merged, as such we'll probably continue using the current code base until it is implemented.

**Comment by mindon** [comment]
another one

https://github.com/BjornTheProgrammer/bun-plugin-html/blob/d621af22e8b4065f4be097cf0cc4c7f3c5984cdf/src/index.ts#L740

may need a fixing for import path prefix before return
```
if ( replacedCount > 0) {
	content = content.replace(
		/(^|\s+)import[^'"]*(['"])(?!['"./]|\w+:\/\/)/g,
		(s) => {
			return s.replace(/(['"])/, '$1./');
		},
	);
}
```

**Comment by BjornTheProgrammer** [comment]
@mindon I appreciate all of the work you have done on `bun-plugin-html`. If you are interested, I would be delighted to make you a maintainer.

**Comment by mindon** [comment]
> @mindon I appreciate all of the work you have done on `bun-plugin-html`. If you are interested, I would be delighted to make you a maintainer.

my pleasure, but i am being busy too :-(  pls focus on your finals first, good luck :-)

**Comment by BjornTheProgrammer** [comment]
Should be fixed in `v2.2.3`
